src/games/supported/DoubleDunk.cpp

```diff
@@ -129,37 +129,79 @@ ModeVect DoubleDunkSettings::getAvailableModes() {
     return modes;
 }
 
+void DoubleDunkSettings::goDown(System &system,
+                            std::unique_ptr<StellaEnvironmentWrapper> &environment) {
+    // this game has a menu that allows to define various yes/no options
+    // this function goes to the next option in the menu
+    unsigned previousSelection = readRam(&system, 0xB0);
+    while(previousSelection == readRam(&system, 0xB0)){
+        environment->act(PLAYER_A_DOWN, PLAYER_B_NOOP);
+        environment->act(PLAYER_A_NOOP, PLAYER_B_NOOP);
+    }
+}
+
+void DoubleDunkSettings::activateOption(System &system, unsigned byteOfInterest,
```

Comment:
unsigned int please.

Reply:
bitOfInterest, not byte. Otherwise the logic below doesn't make sense (e.g. 0x0111 & 0x1100 will give 0x0100).